content/blog/engineering-org-chart-non-technical-founder/index.md

```
@@ -0,0 +1,160 @@
+  image: cover.png
+  og_title: "6.2 · The Org Chart Your Dev Shop Won't Draw"
+  og_description: "The 5-person team your agency pitched is rarely the team writing your code. Six questions to surface who actually ships, who reviews, and who is on-call."
+cover_image_alt: "JetThoughts blog cover showing a redacted org chart with question marks over four boxes and one named senior reviewer"
+canonical_url: "https://jetthoughts.com/blog/engineering-org-chart-non-technical-founder/"
+related_posts: false
+course_nav: false
+---
+
+> **Module 6.2 · Step 1 of 5** · [Tech for Non-Technical Founders 2026](/blog/tech-for-non-technical-founders-2026/) course.
+> Input: a team in place + a signed SOW. Output: a weekly oversight rhythm running by month 3.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)